frontend/src/routes/(authenticated)/user/+page.ts

```diff
@@ -5,8 +5,20 @@ import type {
 
 } from '$lib/gql/types';
 import { getClient, graphql } from '$lib/gql';
-import { invalidate } from '$app/navigation';
-import {refreshJwt} from '$lib/user';
+import { goto, invalidate } from '$app/navigation';
+import { refreshJwt } from '$lib/user';
+import type { PageLoad } from './$types';
+import { browser } from '$app/environment';
+
+export const load = (async ({ url }) => {
+  if (url.searchParams.has('verifiedEmail')) {
+    if (browser) await goto(`${url.pathname}`, { replaceState: true });
+    return { verifiedEmail: true };
+  } else if (url.searchParams.has('changedEmail')) {
+    if (browser) await goto(`${url.pathname}`, { replaceState: true });
+    return { changedEmail: true };
+  }
```

Comment:
could we simplify this to `return {changedEmail: url.searchParams.has('changedEmail'), verirified: [etc...]` it'll keep the branching to a minimum so it's less difficult to understand, maybe just put a comment that both shouldn't be true at once, or use an enum to pick which one you want.